addons/sale/static/src/js/tours/sale.js

```diff
@@ -53,6 +53,10 @@ tour.register("sale_tour", {
     extra_trigger: ".o_sale_order",
     content: _t("Now, we'll create a sample quote."),
     position: "bottom",
+}, {
+    trigger: "h1 span:contains('New')",
+    content: "Check we are on the right page",
+    run: function () {} // check
```

Comment:
Considering it's the onboarding tour of sale, we should avoid adding 'technical' steps to it.
Remove this diff, the little onboarding test you added is already a good improvement for the coverage :).